docs/api-overview.mdx

```diff
@@ -8,6 +8,7 @@ slug: /api-overview
 import TooltipWithDefaults from '../src/components/TooltipWithDefaults';
```

Comment:
Probably want to update the `sidebar_label` (line 4) of this page to be "Documentation" so it is not API -> API

Reply:
Yeah good point. Changing to "reference"